### DISCUSSION TITLE
ShipXplorer AIS Dongle

### DISCUSSION THREAD

**Discussion by OE1MWW** [jvde-github/AIS-catcher#244] [General]
Recently there was an info about the "ShipXplorer AIS Dongle" on rtl-sdr.com
Is available via Amazon (ASIN B09Y61CM12). It has only 3 out of 5 stars in evualtion.
Reason for the 3 out of 5 stars - it's just an RTL dongle with a built in SAW filter for AIS. Some user expected
more and an easy installation. But for the experimenter it seems to perfect and has a reasonable price tag.

Anyone has experience with it ? Sensivity ?

greetings from Vienna / Austria

**Comment by Taxom** [comment]
In this [thread](https://discussions.flightaware.com/t/howto-run-ads-b-receiver-ais-receiver-on-same-rpi/85040/87), there is a discussion about that dongle.

**Comment by jvde-github** [comment]
In my set up I need a filter to get a decent reception and this dongle works well for that (and was cheaper than buying a V3 + AIS filter). Feedback that I have seen as well was that in some cases it does not give an improvement. In general seems to be a safe choice for a dongle for AIS, I guess.

When I bought the dongle there was no mention it was a RTL-SDR or how to install. I think that can be confusing indeed.

**Comment by Gazman429** [comment]
While I'm no big fan of SAW filters as they have a typical 2 to 4 db loss but coupled with a LNA to reverse the losses to balance it out it works.
I bit the bullet and purchased one to try and see how well it works, It is an economical way of dealing with nearby RF interference in the majority of circumstances.
I have 3 VHF commercial trunking sites, 4 VHF high power paging sites, 2 VHF amateur repeaters and local VHF ham operators operating high power nearby and they all caused interference and sensitivity issues of varying degrees.
As I expected the ShipXplorer AIS Dongle didn't improve range but with the built in SAW filter it did reduce or eliminate the interference I was getting from all the nearby VHF interference sources thus improving the decoding of AIS signals with little to no corruption of the AIS data being transmitted compared to a non filtered RTL dongle.
Don't expect range improvements but in noisy RF environments common in urban locations you should see improved decoding quality,
Message rate side by side regular RTL dongle no filter 62 messages / min, ShipExplorer AIS dongle 74 messages / min .
